.jshintrc

```diff
@@ -0,0 +1,53 @@
+{
+  "bitwise": true,
+  "camelcase": true,
+  "curly": false,
+  "eqeqeq": true,
+  "es3": false,
+  "forin": true,
+  "immed": true,
+  "indent": 2,
+  "latedef": "nofunc",
+  "newcap": true,
+  "noarg": true,
+  "noempty": true,
+  "nonew": false,
+  "plusplus": false,
+  "quotmark": true,
+  "regexp": false,
+  "undef": true,
+  "unused": true,
+  "strict": false,
+  "trailing": true,
+  "maxparams": 4,
+  "maxdepth": 2,
+  "maxstatements": 30,
+  "maxcomplexity": 10,
+  "maxlen": 110,
+
+  "asi": false,
+  "boss": false,
+  "debug": false,
+  "eqnull": true,
+  "esnext": true,
```

Comment:
_"This option has been [deprecated](http://jshint.com/docs/options/#esnext) and will be removed in the next major release of JSHint. Use `esversion: 6` instead."_